### ISSUE TITLE
Fade sounds with animations

### ISSUE THREAD

**Issue by Rebelstrike2721** [JannisX11/blockbench#2531]
### Detailed description of your suggestion

Be able to fade sound effects with the animation lengths

**Comment by Rebelstrike2721** [comment]
> Could you explain what you mean by that?

@JannisX11 I mean manage when to fade out a sound effect with the animation

**Comment by JannisX11** [comment]
Do you have a specific use case for that in mind? Since none of the methods of exporting animations in Blockbench currently support sound editing or effects.

**Comment by Rebelstrike2721** [comment]
> Do you have a specific use case for that in mind? Since none of the methods of exporting animations in Blockbench currently support sound editing or effects.

Yes so say that you have a sound effect that you want in sync with an animation there should be a button where you can fade out the sound with the animation if the animation is only played once so then the sound doesnt drag on after the animation is finished thanks for taking the time to respond

**Comment by JannisX11** [comment]
I just did a test in Blockbench, with a sound that is longer than the animation. And Blockbench already stops playing the sound when the animation is ending.

**Comment by Rebelstrike2721** [comment]
Oh ok so its alrwady implementes then how did you do it if you dont mind me saying like does this happen with loop as well as in it starts the sound back at the beginning
